packages/table/tests/__snapshots__/Table.test.tsx.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This snapshot does not look right.

Reply:
Agreed, I really wonder why though...

Reply:
My guess only cells in the viewport get rendered.

Reply:
yeah, that could be right 🤔

Reply:
I will try to figure out a way to solve that issue.